RealmSwift/Tests/ObjectSchemaTests.swift

```diff
@@ -24,7 +24,9 @@ class ObjectSchemaTests: TestCase {
 
     override func setUp() {
         super.setUp()
-        objectSchema = Realm().schema["SwiftObject"]
+        autoreleasepool {
+            self.objectSchema = Realm().schema["SwiftObject"]
```

Comment:
This is not meaningful code duplication, and it makes the rest of the code less readable to other developers.